src/feature_sections.rs

```diff
@@ -280,3 +280,41 @@ impl PmuMappings {
         Ok(Self(vec.into_iter().collect()))
     }
 }
+
+/// The clock data header contains information about the clock used to
+/// record timestamps in the perf.data file.
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct ClockData {
+    /// Version of this structure, currently 1.
+    pub version: u32,
+    /// Clock ID used for timestamps
+    pub clockid: u32,
+    /// Wall clock time in nanoseconds.
+    pub wall_clock_ns: u64,
```

Comment:
Do you know what this value is reference to? Maybe the unix epoch?

Reply:
Yes, it is indeed since the epoch. I will update the doc comment